site/slices/contact-us-slice/form/index.js

```diff
@@ -15,6 +15,23 @@ class Form extends Component {
     };
   }
 
+  componentDidMount = () => {
+    const { stateNavigator } = this.context;
+    stateNavigator.onNavigate(this.focusText);
+  }
+
+  componentWillUnmount = () => {
+    const { stateNavigator } = this.context;
+    stateNavigator.offNavigate(this.focusText);
+  }
```

Comment:
No problem, thanks for the pointer. I've changed things around and removed the `stateNavigator` from the components.